daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -985,6 +985,9 @@ private[lf] object SBuiltin {
           coid,
           crash(s"Contract ${coid.coid} is missing from cache"),
         )
+      onLedger.ptx.consumedBy
```

Comment:
I think I don’t quite understand why this is the right place for this check. Shouldn’t we check this in `SBUFetch` instead of `SBUBeginExercise`? I’m actually not sure where we are checking it for fetches now given that you dropped `mustBeActive`.

Reply:
My mistake. We just need the one for fetch.